src/models/exit-strategy-option.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const colors = require('colors/safe');
+const Logger = require('../logger.js');
+
+function getExitStrategy (follow, exitOnDeploy) {
+  if (follow) {
+    if (exitOnDeploy === 'deploy-start') {
+      throw new Error('The `follow` and `exit-on` set to "deploy-start" options are not compatible');
+    }
+    Logger.println(`${colors.yellow('The `follow` option is deprecated and will be removed in an upcoming major, use --exit-on set to "never" instead')}`);
```

Comment:
nitpick: no need to use a template string which only contains a hole with a string. You could do:

```suggestion
    Logger.println(colors.yellow('The `follow` option is deprecated and will be removed in an upcoming major, use --exit-on set to "never" instead'));
```